lost/src/test/java/com/mapzen/android/lost/internal/LostApiClientImplTest.java

```diff
@@ -29,7 +29,7 @@
 public class LostApiClientImplTest {
   private LostApiClient client;
 
-  @Before public void setUp() throws Exception {
```

Comment:
Setup and test methods in JUnit should all include `throws Exception` by convention.

This makes the tests less brittle since it ensures that if any method signatures change for the production code under test to throw an exception then the tests will still compile.